pkg/utils/ebpf2graph/graph.go

```diff
@@ -0,0 +1,172 @@
+package graphutils
```

Comment:
The go convention is that the main file of a package is named with the same name of the package and folder. So, please align: `ebpf2graph`, `graph.go` and `graphutils`.